install.sh

```diff
@@ -187,5 +200,30 @@ function build_codemirror {
 
 run $BUILD/CodeMirror build_codemirror
 
+# Install Android SDK
+
+run $DOWNLOADS        wget https://dl.google.com/android/repository/sdk-tools-linux-3859397.zip
+run $BUILD            unzip $DOWNLOADS/sdk-tools-linux-3859397.zip
```

Comment:
This would be cleaner:

    run $BUILD             mkdir -p Android/Sdk
    run $BUILD/Android/Sdk unzip $DOWNLOADS/sdk-tools-linux-3859397.zip